podcasts/TranscriptFormat.swift

```diff
@@ -36,6 +49,11 @@ public enum TranscriptFormat: String {
 
 extension Episode.Metadata.Transcript {
     public var transcriptFormat: TranscriptFormat? {
-        return TranscriptFormat(rawValue: self.type)
+        for format in TranscriptFormat.allCases {
```

Comment:
Nit:
I think you can either filter the for using `where`
```
for format in TranscriptFormat.allCases where format.possibleTypes.contains(type) {
            return format
        }
        return nil
```

or just `return TranscriptFormat.allCases.first { $0.possibleTypes.contains(type) }`

Reply:
Done here: https://github.com/Automattic/pocket-casts-ios/pull/2193/commits/29f14f6563dfed79ab621953fb41ea63b5aa0e8a